dat/Atomiswave.dat

```diff
@@ -0,0 +1,76 @@
+clrmamepro (
+	name "Atomiswave"
+	description "Atomiswave"
+	version 2021.7.9
+	author "libretro"
+)
+
+game (
+	name "Metal Slug 6"
+	description "Metal Slug 6"
+	genre "Action"
+	rom ( name mslug6.zip size 128344 crc 3582912721 131422139 md5 17b550ce0b58778d3ac304801be03768 sha1 637811d275a2716cd9abfe1d794bece650b65736 )
```

Comment:
Are these .zip files torrentzipped? .zip files can have variable sizes.

Reply:
The size is wrong on that one, it is a zip file at 131.4  MegaBytes. I will correct it with the following "131422139" I might as well review the ones I did again for Atomiswave as well so there's no mistakes.